packages/react/src/Logo/Logo.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes } from 'react'
+import { LogoAmsterdam, LogoGgdAmsterdam, LogoStadsarchief, LogoStadsbankVanLening, LogoVgaVerzekeringen } from '.'
+
+type LogoBrand = 'amsterdam' | 'ggd-amsterdam' | 'stadsarchief' | 'stadsbank-van-lening' | 'vga-verzekeringen'
+
+export interface LogoProps extends HTMLAttributes<HTMLElement> {
+  brand?: LogoBrand
+}
+
+const logoConfig: Record<LogoBrand, Function> = {
+  amsterdam: LogoAmsterdam,
+  'ggd-amsterdam': LogoGgdAmsterdam,
+  stadsarchief: LogoStadsarchief,
+  'stadsbank-van-lening': LogoStadsbankVanLening,
+  'vga-verzekeringen': LogoVgaVerzekeringen,
+}
+
+export const Logo = forwardRef(
+  ({ brand = 'amsterdam', className, ...restProps }: LogoProps, ref: ForwardedRef<HTMLElement>) => {
+    const LogoComponent = logoConfig[brand]
+
+    return (
+      <span {...restProps} ref={ref} className={clsx('amsterdam-logo', className)}>
```

Comment:
Do we need the span here? We could just set all classes and props directly on the svg, no? This has the added benefit that the user can access the svgs attributes (`alt`, etc.), which they can't now

Reply:
Do you mean having the svg tag here and putting the paths in `LogoBrand`? It could be done but I have a bunch of complaints from typescript concerning ref and brand as props. Can you make a suggestion? 

It was originaly a span to hold an `<img>` tag and have it scale with the page menu and match the line-height. Something that's also easier keeping the span?

Reply:
Ok, it has been changed. Only thing I wasn't able to get going was the ForwardRef unit test. So I skipped it.

Reply:
I added a commit to pass the refs to the LogoComponents